vpr/src/place/move_utils.h

```diff
@@ -57,4 +51,8 @@ bool find_to_loc_uniform(t_logical_block_type_ptr type,
                          float rlim,
                          const t_pl_loc from,
                          t_pl_loc& to);
+
+//Accessor and setter for f_placer_debug
```

Comment:
Explain what setting this to true or false means.  (idea: person who wants to call these should only have to read the .h file.

Reply:
Is placer_debug_enabled the right name?  Or should it be placer_breakpoint_reached? (or something like that).